data/avro-schema/src/main/resources/avro/net/corda/data/flow/FlowStartContext.avsc

```diff
@@ -43,7 +43,18 @@
       "name": "startArgs",
       "type": ["null", "string"],
       "default": null,
-      "doc": "The body of the request provided when the flow was started if this flow was started via RPC. Null otherwise"
+      "doc": "The body of the request provided when the flow was started if this flow was started via RPC. Null otherwise."
+    },
+    {
+      "name": "contextPlatformProperties",
+      "type": ["null",
+        {
+          "type": "map",
+          "values": "string"
+        }
+      ],
```

Comment:
Can probably be formatted better.

Reply:
I'll run it through a formatter, the IDE wont do this for me unfortunately.

Reply:
There is a plugin for `avsc` files that I'm using which handles formatting. Intellij recommended it when I opened an `avsc` file.